.github/workflows/post-release.yml

```diff
@@ -71,3 +72,9 @@ jobs:
                     asset_path: ./build/installer.sh
                     asset_name: installer.sh
                     asset_content_type: text/x-shellscript
+
+            # Trigger Buddy CI jobs
+            -   name: Trigger CI pipeline
```

Comment:
Why did you choose to adjust "create/publish installer" job instead of making a separate one or adjusting "docker build" job?